website/docs/integrator-section/transferable-record/preparing-transferable-record.md

```diff
@@ -3,36 +3,37 @@ id: preparing-transferable-record
 title: Preparing Transferable Record
 sidebar_label: Preparing Transferable Record
 ---
+import WrapSingleDocument from "/src/reusables/_wrap-single-document.mdx";
 
 In this section, we will prepare the DNS and the content of the transferable record to be issued.
 
-## DNS Configuration
+## DNS configuration
 
 Similar to [binding the document store to a domain name](/docs/integrator-section/verifiable-document/ethereum/document-store), you will have to bind the identity of the token registry to a domain name.
 
-To do that simply create a `TXT` record on your domain with the following entry:
+To do that, create a `TXT` record on your domain with the following entry value:
 
-```txt
-openatts net=ethereum netId=3 addr=0x8431012Bc040942B59e3C5bf428221eab0b2f723
-```
+| Type | Domain      | Entry Value                                                     |
+| ---- | ----------- | --------------------------------------------------------------- |
+|`TXT` | example.com | "openatts net=`ethereum` netId=`11155111` addr=`0x8431012Bc040942B59e3C5bf428221eab0b2f723`" |
 
-You will need to replace the token registry address `0x8431012Bc040942B59e3C5bf428221eab0b2f723` with the address you've got from the previous step.
+You will need to replace the token registry address `0x8431012Bc040942B59e3C5bf428221eab0b2f723` with the address you get from the previous step.
 
-If you like to use our sandbox DNS for the purpose of the exercise, you may instead run the following command, following the instructions from [earlier](/docs/integrator-section/verifiable-document/ethereum/dns-proof):
+If you want to use our sandbox DNS for the purpose of the exercise, run the following command instead following the instructions from [earlier](/docs/integrator-section/verifiable-document/ethereum/dns-proof):
 
 ```sh
-open-attestation dns txt-record create --address 0x8431012Bc040942B59e3C5bf428221eab0b2f723 --network-id 3
+open-attestation dns txt-record create --address 0x8431012Bc040942B59e3C5bf428221eab0b2f723 --network-id 11155111
 ```
 
-If you like more detailed setup instructions, you may refer to the [documentation for configuring DNS](/docs/developer-section/quickstart/configure-dns).
+If you want to view more detailed setup instructions, see the [documentation for configuring DNS](/docs/developer-section/quickstart/configure-dns).
```

Comment:
This links to the Developer section which may cause some confusion, should we make this open in a new tab? That way the user can still come back to this page.

Reply:
Tried to add `target="_blank"` at the end of the link: 

`[documentation for configuring DNS](/docs/developer-section/quickstart/configure-dns){:target="_blank"}`

But it couldn't work properly in browser: 

> If you want to view more detailed setup instructions, see the [documentation for configuring DNS](http://localhost:3000/docs/developer-section/quickstart/configure-dns){:target="_blank"}.

I will save this as a future item, and have to keep the link as is for the interim.